policycoreutils/scripts/chcat: quote file names given as arguments

vmojzis opened the pull request:
Add quotes around filenames that are passed to getstatusoutput()
in order for special characters and spaces to be handled correctly.

Fixes: https://bugzilla.redhat.com/show_bug.cgi?id=1013774

bachradsusi commented on policycoreutils/scripts/chcat:
```diff
@@ -91,13 +88,16 @@ def chcat_user_add(newcat, users):
+                subprocess.check_call(["semanage", "login", "-m", "-r", new_serange, "-s", user[0], u],
+                                        stderr=subprocess.STDOUT,
+                                        shell=False)
+        except subprocess.CalledProcessError as e:
```
What about:

    if add_ind:
        cmd = ["semanage", "login", "-a", "-r", new_serange, "-s", user[0], u]
    else:
        cmd = ["semanage", "login", "-m", "-r", new_serange, "-s", user[0], u]
    try:
        subprocess.check_call(cmd, ...)

bachradsusi commented:
Please rebase

vmojzis commented:
Rebased.

bachradsusi commented:
https://github.com/fedora-selinux/selinux/pull/38#discussion_r149412116 is still valid. Could you please change it so that `check_call` is used only once on `cmd`?

vmojzis commented:
- Rebased
- fixed [#38 (comment)](https://github.com/fedora-selinux/selinux/pull/38#discussion_r149412116)  
- replaced string.join() with ",".join() so that the code works with python3

bachradsusi commented:
https://github.com/fedora-selinux/selinux/commit/db648666ab65f9efff7f1ba331c8e337420b1cd4
